acceptance/Rakefile

```diff
@@ -17,6 +17,73 @@ EOS
       ENV['TESTS'] = get_test_sample.join(",")
       Rake::Task["ci:test:aio"].invoke
     end
+
+    desc <<-EOS
+Run tests on docker quickly and easily. The docker container is set up to mount
+your puppet directory in the container. This means you can edit code or test
+files and rerun tests without reconfiguring your test environment.
+
+Defaults to running all tests unless TESTS is set. TESTS is a comma seperated
+list of test files to run.
+
+  $ bundle exec rake ci:test:docker TESTS='tests/puppet_apply_basics.rb,tests/resource/file/content_attribute.rb'
```

Comment:
not sure why `puppet _apply_basics` tries to run `puppet master --help`. Since we don't have a master, maybe choose  a different agent-only test (or remove that)?

Reply:
I changed this to not reference actual tests